rtb/norme-di-progetto/src/gestione-configurazione.tex

```diff
@@ -1,11 +1,50 @@
+\pagebreak
+\pagebreak
 \subsection{Gestione della Configurazione}
 
 \subsubsection{Descrizione, scopo ed aspettative}
+    Lo scopo è di supervisionare e regolare in modo organizzato la creazione di 
+    documenti e codice. Ogni elemento soggetto a configurazione sarà soggetto a versionamento
+    e controllo delle modifiche, al fine di assicurare l'integrità del prodotto 
+    nel tempo.
+    \\
+    Tutti gli strumenti dedicati alla configurazione utilizzati per la produzione
+    di documenti e codice vengono raccolti, organizzati e coordinati. Questo include
+    la gestione della struttura e della disposizione dei file all'interno del repository, 
+    nonché gli strumenti per il versionamento e il coordinamento.
 
 \subsubsection{Versionamento}
-
+Ogni modifica apportata a un documento genera una nuova versione seguendo il formato
+\textbf{v[X].[Y].[Z]}, dove:
+\begin{itemize}
+    \item \textbf{X:} Rappresenta la versione approvata dal \emph{Responsabile}, l'unico autorizzato ad incrementarla;
+    \item \textbf{Y:} Rappresenta la versione approvata dal \emph{Verificatore}, l'unico autorizzato ad incrementarla;
+    \item \textbf{Z:} Rappresenta la versione dell'ultima modifica.
+\end{itemize}
+Ogni parte del codice di versione inizia da 0 e ritorna a 0 ogni volta che la componente alla sua sinistra
+viene incrementata.
 \subsubsection{Repository}
-
-    \subsubsubsection{Tecnologie}
-
-    \subsubsubsection{Struttura Repository}
\ No newline at end of file
+    \subsubsubsection{Tecnologie e strumenti}
+    Per il progetto viene usato \emph{GitHub}, che a sua volta implementa il sistema di versionamento 
+    \emph{Git}. Il gruppo fa pieno uso delle funzionalità di \emph{GitHub} tra cui:
+    \begin{itemize}
+        \item \textbf{Issue:} Permette di discutere, tracciare e risolvere un problema nel progetto;
+        \item \textbf{Milestone:} Permette di raggruppare e monitorare insieme una serie di issues aiutando a tracciare il progresso verso obiettivi del progetto;
+        \item \textbf{Pull request:} Consente di proporre delle modifiche al codice e di richiederne la revisione da un altro membro.
+    \end{itemize}
+    \subsubsubsection{Struttura Repository}
+    Si è deciso di utilizzare un'unica repository contenente il codice sorgente per documentazione e \emph{PoC}.
+    All'interno della repository troviamo una cartella per ogni documento denominata \emph{nome-documento}. Inoltre la
```

Comment:
In realtà non è proprio così, infatti la repo è composta da 3 cartelle così divise: 

- Template: contenente il template per il documento latex;
- Candidatura: contenente i documenti relativi alla candidatura. Aggiungere inoltre quali documenti contiene questa cartella (i nomi dei documenti).
- RTB: contenente i documenti relativi alla revisione. Aggiungere inoltre da quali cartelle è formato (l'idea sarebbe di suddividere in documentazione interna e documentazione esterna, e fare una lista di tutti i documenti contenuti in ognuno di esse).

Ricordarsi di nominare anche i verbali.